packages/commons/src/auth/jwk.ts

```diff
@@ -24,10 +25,15 @@ export const calculateKid = (jwk: JsonWebKey): string => {
 };
 
 function createPublicKey(key: string): KeyObject {
+  const pemKey = decodeBase64ToPem(key);
   try {
```

Comment:
on a side note, for future readers (and future us) may you add a comment explaining the little trick of using both `createPrivateKey` and `createPublicKey`?